### PR TITLE
add base documentation for all operator+config resources

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
/assign @kalexand-rh 

bump

**Comment by vikram-redhat** [comment]
@xltian - could you please get someone to give QE approval for this?

**Comment by xltian** [comment]
@jianzhangbjz can you please review? thanks

**Review by sdodson — APPROVED** [review]
LGTM

**Comment by bparees** [comment]
@xltian @jianzhangbjz bump

**Comment by jianzhangbjz** [comment]
@xltian @bparees Sorry for the late to reply. LGTM based on my understanding.
 @jianlinliu Since it's related to the installation mainly, could you help double-review this document prevent me missing something else? Thanks!

**Comment by jianlinliu** [comment]
From the PR, most of description is related to all kinds of operators, nothing with installer, I can not provide much information here. After a glance, LGTM.

**Comment by bparees** [comment]
@vikram-redhat @kalexand-rh sounds like this is good to go.

File: installing/install_config/customizations.adoc [hunk 1]
```diff
@@ -5,91 +5,152 @@ include::modules/common-attributes.adoc[]
...
-|Configure the
-xref:../../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[internal OAuth server].
+|===
 
-|Project
-|Configure xref:../../applications/projects/configuring-project-creation.adoc#configuring-project-creation[how projects are created].
+[id="operator-configuration-resources_{context}"]
+== Operator configuration resources
+
+These configuration resources are cluster scoped singletons which control behavior of a specific component as
```
Comment (mburke5678) [inline installing/install_config/customizations.adoc]: s/cluster scoped/cluster-scoped
Is there a better word for _singletons_? It is not a term we use, likely to _techie_ or _jargony_

File: installing/install_config/customizations.adoc [hunk 2]
```diff
@@ -5,91 +5,152 @@ include::modules/common-attributes.adoc[]
...
+owned by a particular operator.  All resources are cluster scoped and the instances are named `cluster`.
```
Comment (mburke5678) [inline installing/install_config/customizations.adoc]: s/cluster scoped/cluster-scoped

File: installing/install_config/customizations.adoc (hunk 1 above)
Comment (bparees) [reply installing/install_config/customizations.adoc]: The concept here is that there is a single instance of this resource type for the entire cluster.  Open to suggestions.

Comment (jianzhangbjz) [reply installing/install_config/customizations.adoc]: I think `cluster-scoped instance` is OK.

**Comment by vikram-redhat** [comment]
@bparees looks like one change requested for cluster scoped to cluster-scoped. When you will update that, I will merge and CP. Thanks for submitting this.

File: installing/install_config/customizations.adoc (hunk 1 above)
Comment (bparees) [inline installing/install_config/customizations.adoc]:
```suggestion
These configuration resources are cluster-scoped instances which control behavior of a specific component as
```

File: installing/install_config/customizations.adoc (hunk 2 above)
Comment (bparees) [reply installing/install_config/customizations.adoc]:
```suggestion
owned by a particular operator.  All resources are cluster-scoped and the instances are named `cluster`.
```

**Comment by bparees** [comment]
@vikram-redhat change requests applied.

**Comment by bparees** [comment]
@openshift/team-documentation I think you guys are the perfect candidate for github's suggested changes feature:
https://twitter.com/github/status/1053360435074949120?lang=en

saves the PR author time since they don't have to go make the edits you are proposing, repush the PR, etc.

**Comment by vikram-redhat** [comment]
@bparees nice suggestion and I will bring it up with the team. Thanks for making the fix. I will merge and CP.

**Comment by mburke5678** [comment]
@bparees The Custom Resources section that you edited is currently commented-out and doesn't appear. Should these resources be exposed in the docs?
https://github.com/openshift/openshift-docs/pull/15905/files#diff-162300a9be635b34c5f6aeef7585d971R160

**Comment by bparees** [comment]
> @bparees The Custom Resources section that you edited is currently commented-out and doesn't appear. Should these resources be exposed in the docs?

no, that content should just be deleted at this point.  i'm not sure why it was put in as commented out in the first place.

**Comment by bparees** [comment]
or you can leave them commented out and as teams decide to go document their operator-specific resources they can uncomment them.

**Comment by mburke5678** [comment]
Editorial follow up: https://github.com/openshift/openshift-docs/pull/16143

**Comment by kalexand-rh** [comment]
@mburke5678, I made the table earlier in the 4.1 release and removed it after some feedback from Derek. It wasn't clear if it would be required later.

**Comment by abhinavdahiya** [comment]
/retitle Bug 1729308: add base documentation for all operator+config resources

**Pull request merged**